src/libraries/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -1621,6 +1623,13 @@ private static HttpClient CreateHttpClient(HttpClientParameters parameters, Http
                     handler.UseCookies = false;
                 }
 
+                if (parameters.ServicePoint is { } servicePoint)
+                {
+                    handler.MaxConnectionsPerServer = servicePoint.ConnectionLimit;
+                    handler.PooledConnectionIdleTimeout = servicePoint.MaxIdleTime == -1 ? Threading.Timeout.InfiniteTimeSpan : TimeSpan.FromMilliseconds(servicePoint.MaxIdleTime);
+                    handler.PooledConnectionLifetime = servicePoint.ConnectionLeaseTimeout == -1 ? Threading.Timeout.InfiniteTimeSpan : TimeSpan.FromMilliseconds(servicePoint.ConnectionLeaseTimeout);
```

Comment:
If the web request is creating its own dedicated handler for this request, are any of these actually going to matter?

Reply:
No, it will not matter now, it will only be useful after we change `HttpClient` cache mechanism in `HttpWebRequest`, but we can implement them later with those change as well.